lib/tasks/enrollment_renewal_eligibility_report.rake

```diff
@@ -0,0 +1,52 @@
+desc "Run a report for enrollment eligibility"
+task :enrollment_renewal_eligibility_report => :environment do
+  file_name = "#{Rails.root}/enrollment_eligibility_report_#{TimeKeeper.date_of_record.strftime('%Y_%m_%d')}.csv"
+  field_names = %w[EnrollmentHbxId EnrollmentAASMState PrimaryHbxID MemberHbxID MemberFirstName MemberLastName MemberDOB member-is_applying_coverage member-citizen_status member-is_incarcerated member-state member_present_in_manage_family? applicant-is_applying_coverage applicant-citizen_status applicant-is_incarcerated applicant-state error_message]
+  CSV.open(file_name, 'w', force_quotes: true) do |csv|
+    csv << field_names
+    active_enrollments = HbxEnrollment.where(:aasm_state.nin => %w[shopping coverage_canceled coverage_terminated coverage_expired])
```

Comment:
Confirmed that the intended purpose of this script is to report specifically on IVL enrollments (including Coverall enrollments).  We can update this query to limit the enrollments like so:
```Ruby
active_enrollments = HbxEnrollment.where(:kind.in => %w[individual coverall], :aasm_state.nin => %w[shopping coverage_canceled coverage_terminated coverage_expired])
```